codegen_glibmm/templates/stub.h.templ

```diff
@@ -86,10 +86,15 @@ protected:
 private:
     bool emitSignal(const std::string &propName, Glib::VariantBase &value);
 
+    class RegisteredObject {
```

Comment:
struct for POD? (No prior POD in this code... but struct for POD is a common convention at least. :)

Reply:
Both are fine to me. There are people that think that `struct` should never be used in C++ code, so I was being cautious :-)

I'll change this. :-)